### ISSUE TITLE
Issue Report: Setting root to "savePath" in the code-workspace file results in a warning

### ISSUE THREAD

**Issue by kurikimuchi** [glenn2223/vscode-live-sass-compiler#282]
Can savePath be set to root?
If so, I would like the warning message to not be displayed.

### To Reproduce
1. create `.code-workspace` file
2. set `"/"` to `"savePath"`
3. **Warning message and yellow wavy line appears** when saving the `.code-workspace` file

note: **But css files are generated** in root with no problem

### Warning Message
```
Must start with any of:
`/` or `\` (for workspace root)
`~/` or `~\` for relative to the file being processed. Must not end with a path separator (`/` or `\`)
Set the save location of exported CSS.

Set the relative path from Workspace Root.

'/' stands for your workspace root.

Example: /subfolder1/subfolder2. (NOTE: if folder does not exist, folder will be created as well).
```

### Environment
- Windows 10
- Live Sass Compiler v6.0.3

### For Reference
↓ `.code-workspace` file
![githubex1](https://user-images.githubusercontent.com/42775886/227940729-5a675e9a-de01-4edd-991e-86f9467a23bf.png)

Best Regards.

**Comment by glenn2223** [comment]
Thanks for pointing out this error, I will tweak the warning regex - you are fine to continue as is unit I can sort the issue. Thanks again

**Comment by kurikimuchi** [comment]
Sorry for posting the same post over and over again due to the glitch.
Thank you for your quick response.

**Comment by QuentiumYT** [comment]
Hello, sorry to not open a new issue, but I'm just asking if this was a correct warning or still an issue with version 6.0.4. Thanks for your reply.

![image](https://user-images.githubusercontent.com/35229942/230786925-74eac10f-3613-4047-90ea-eb6c4109f408.png)

**Comment by glenn2223** [comment]
> Hello, sorry to not open a new issue, but I'm just asking if this was a correct warning or still an issue with version 6.0.4. Thanks for your reply.

You're right, I made a slight error in v6.0.4. There was originally a `?`, which I changed to a `+` - when it should have been a `*`. Fixed in v6.0.5 - coming imminently

**Comment by QuentiumYT** [comment]
Thank you very much! :)
